docker-compose.override.yml.DISABLED

```diff
@@ -0,0 +1,92 @@
+version: "3.7"
```

Comment:
Oops, this file shouldn't have been committed at all. It doesn't have anything sensitive in it but we don't need it in the repo.